### PR TITLE
8.0: update imagestreams and README.

### PR DESCRIPTION
**Pull request by tmds**

### CODE REVIEW

File: dotnet_imagestreams.json [hunk 1]
```diff
@@ -25,20 +25,62 @@
...
+                            "name": "registry.access.redhat.com/ubi8/dotnet-80:8.0"
+                        }
+                    },
+                    {
+                        "name": "8.0",
+                        "annotations": {
+                            "openshift.io/display-name": ".NET 8 (UBI 8)",
+                            "description": "Build and run .NET 8 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/redhat-developer/s2i-dotnetcore/tree/main/8.0/build/README.md.",
+                            "iconClass": "icon-dotnet",
+                            "tags": "builder,.net,dotnet,dotnetcore,dotnet80,hidden",
```
Comment (omajid) [inline dotnet_imagestreams.json:71]: Is the `hidden` tag intentional?

**Review by omajid — APPROVED** [review]
LGTM!

**Comment by omajid** [comment]
It would be nice to have some sort of CI to verify that the changes still result in valid imagestreams.

**Comment by tmds** [comment]
> It would be nice to have some sort of CI to verify that the changes still result in valid imagestreams.

Because I only do this once a year, I manually import these in OpenShift for verification.
I like going through the steps and seeing how it behaves on OpenShift.

**Comment by omajid** [comment]
> Because I only do this once a year, I manually import these in OpenShift for verification.
>
>I like going through the steps and seeing how it behaves on OpenShift.

Oh, that's great! I didn't know you did that.

**Pull request merged**